curvefs/src/client/rpcclient/channel_manager.h

```diff
@@ -72,11 +72,6 @@ ChannelManager<T>::GetOrCreateChannel(const T &id,
     }
 
     curve::common::WriteLockGuard guard(rwlock_);
-    auto iter = channelPool_.find(id);
```

Comment:
First add a read lock to judge whether the channel exists; add a write lock, and then judge again, because it is possible that other threads have already added the channel like:
<img width="444" alt="image" src="https://github.com/opencurve/curve/assets/15689619/fa20c37b-f5fb-43d0-9fb9-840b81b2110f">